rpcserver.go

```diff
@@ -3459,34 +3458,38 @@ func UnmarshalOutpoint(outpoint string) (*wire.OutPoint, error) {
 // unmarshalLeafKey un-marshals a leaf key from the RPC form.
 func unmarshalLeafKey(key *unirpc.AssetKey) (universe.LeafKey, error) {
 	var (
-		leafKey universe.LeafKey
-		err     error
+		leafKey        universe.LeafKey
```

Comment:
Actually, I think we should do the opposite here and just declare the `leafKey` here, since that's all we're interested in after the `switch`. All other variables can be inline (which solves the `err` shadowing too, as that wouldn't be declared here anymore either).